Prace_domowe/Praca_domowa4/Wolny_Krzysztof/PD4.ipynb

```diff
@@ -0,0 +1,1967 @@
+{
```

Comment:
<p>Fajnie jakbyś pokazał rozkład tej zmiennej, bo ciężko powiedzieć czy to dużo, czy mało</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>